src/builder.rs

```diff
@@ -365,7 +365,6 @@ impl<W: Write> Builder<W> {
     /// # Examples
     ///
     /// ```
-    /// use std::fs;
```

Comment:
I also saw that Clippy isn't running in CI, and there are quite a few warnings: https://gist.github.com/schneems/fd26378009f9ac28ef282226de6ba5e7. Before going overboard, I wanted to send this one small change. In searching history to see if that's intentional, I saw https://github.com/alexcrichton/tar-rs/pull/79#discussion_r71975483 where you prefer a different style than Clippy suggests (which makes me feel that you would be opposed to adding a clippy check to CI).